pkg/cmd/eventlistener/list_test.go

```diff
@@ -26,6 +26,7 @@ import (
 	cb "github.com/tektoncd/cli/pkg/test/builder"
 	"github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
 	triggertest "github.com/tektoncd/triggers/test"
+	el "github.com/tektoncd/triggers/test/builder"
```

Comment:
`"github.com/tektoncd/triggers/test/builder"` is already imported as `tb`, so just use `tb` instead of using a different alias (`el`).